src/main/java/hudson/plugins/sonar/SonarInstallation.java

```diff
@@ -28,6 +28,7 @@
 import hudson.util.Secret;
 import java.io.Serializable;
 import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
```

Comment:
Note that as of https://github.com/jenkinsci/jenkins/pull/4604 we prefer SpotBugs annotations to JSR 305.